packages/test/local-server-tests/src/test/stagingMode.spec.ts

```diff
@@ -5,12 +5,14 @@
 
 import { strict as assert } from "assert";
 
+import { generatePairwiseOptions } from "@fluid-private/test-pairwise-generator";
```

Comment:
I split out most of the changes in this file into #24535 as I was initially thinking the rest of this PR would be trivial, but it's gotten big enough now I want to make it more obvious to reviewer which parts are actually adding coverage :)